src/test/java/com/iexec/common/sdk/order/payload/RequestOrderTest.java

```diff
@@ -0,0 +1,67 @@
+package com.iexec.common.sdk.order.payload;
+
+import com.iexec.common.chain.DealParams;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.Map;
+
+class RequestOrderTest {
```

Comment:
Can we add a test that serializes all params at once?

Reply:
Fixed in https://github.com/iExecBlockchainComputing/iexec-common/pull/298/commits/7107a8db754fd4807bf1ec149afd78fa41506bd1